Source/ZoomNet.UnitTests/WebhookParserTests.cs

```diff
@@ -344,6 +415,76 @@ public void MeetingServiceIssue()
 			parsedMeeting.Timezone.ShouldBe("America/Los_Angeles");
 		}
 
+		[Fact]
+		public void RecordingCompleted()
+		{
+			var parsedEvent = (RecordingCompletedEvent)new WebhookParser().ParseEventWebhook(RECORDING_COMPLETED_WEBHOOK);
+
+			parsedEvent.EventType.ShouldBe(EventType.RecordingCompleted);
+			parsedEvent.Timestamp.ShouldBe(1626230691572.FromUnixTime(Internal.UnixTimePrecision.Milliseconds));
+			parsedEvent.DownloadToken.ShouldBe("abJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJodHRwczovL2V2ZW50Lnpvb20udXMiLCJhY2NvdW50SWQiOiJNdDZzdjR1MFRBeVBrd2dzTDJseGlBIiwiYXVkIjoiaHR0cHM6Ly9vYXV0aC56b29tLnVzIiwibWlkIjoieFp3SEc0c3BRU2VuekdZWG16dnpiUT09IiwiZXhwIjoxNjI2MTM5NTA3LCJ1c2VySWQiOiJEWUhyZHBqclMzdWFPZjdkUGtrZzh3In0.a6KetiC6BlkDhf1dP4KBGUE1bb2brMeraoD45yhFx0eSSSTFdkHQnsKmlJQ-hdo9Zy-4vQw3rOxlyoHv583JyZ");
+			parsedEvent.AccountId.ShouldBe("AAAAAABBBB");
+			parsedEvent.Recording.ShouldNotBeNull();
+			parsedEvent.Recording.Id.ShouldBe(1234567890);
+			parsedEvent.Recording.Uuid.ShouldBe("4444AAAiAAAAAiAiAiiAii==");
+			parsedEvent.Recording.HostId.ShouldBe("x1yCzABCDEfg23HiJKl4mN");
+			parsedEvent.Recording.AccountId.ShouldBe("x1yCzABCDEfg23HiJKl4mN");
+			parsedEvent.Recording.Topic.ShouldBe("My Personal Recording");
+			parsedEvent.Recording.StartTime.ShouldBe(new DateTime(2021,7,13,21,44,51, DateTimeKind.Utc));
+			parsedEvent.Recording.Password.ShouldBe("132456");
+			parsedEvent.Recording.Duration.ShouldBe(60);
+			parsedEvent.Recording.ShareUrl.ShouldBe("https://example.com");
+			//parsedEvent.Recording.RecordingType.ShouldBe(RecordingType.Personal); // property not implemented
```

Comment:
Thanks for pointing this out. I'll raise an issue to track the fact that this field is missing and should be added

Reply:
This is more complex because the library already has a type `REcordingType` used for something else - inconsistencies in the Zoom API I guess

Reply:
I agree, there's a name conflict. Let's continue this discussion in #298